src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -162,17 +180,23 @@ private void Stop()
         {
             logger.LogDebug(Resources.INFO_TerminatingServer);
 
+            if (Process != null)
```

Comment:
Wasn't this a memory leak? we have another one in the cfamily ProcessRunner.

Reply:
Why - because the event handlers weren't explicitly unregistered?

It shouldn't have been; the only thing that has a reference to the Process instance is this class, so once this class sets its `Process` variable to null the process instance is unreachable and eligible for collection.
The fact that the process instance holds references to the `EslintBridgeProcess` won't stop the process instance from being collected. It doesn't hurt to explicitly unregister the event handlers but it isn't strictly necessary here.

The only occasion where it does matter is if the publisher lives longer than the subscriber, in which case the event registrations can keep the subscriber alive.

Or did you mean something else?